scripts/zones/Reisenjima/IDs.lua

```diff
@@ -15,10 +15,12 @@ zones[xi.zone.REISENJIMA] =
         GIL_OBTAINED                  = 6391, -- Obtained <number> gil.
         KEYITEM_OBTAINED              = 6393, -- Obtained key item: <keyitem>.
         ITEMS_OBTAINED                = 6399, -- You obtain <number> <item>!
+        NOTHING_OUT_OF_ORDINARY       = 6404, -- There is nothing out of the ordinary here.
         CARRIED_OVER_POINTS           = 7001, -- You have carried over <number> login point[/s].
         LOGIN_CAMPAIGN_UNDERWAY       = 7002, -- The [/January/February/March/April/May/June/July/August/September/October/November/December] <number> Login Campaign is currently underway!
         LOGIN_NUMBER                  = 7003, -- In celebration of your most recent login (login no. <number>), we have provided you with <number> points! You currently have a total of <number> points.
         MEMBERS_LEVELS_ARE_RESTRICTED = 7023, -- Your party is unable to participate because certain members' levels are restricted.
+        YOU_HAVE_USED                 = 7609, -- You have used %.
```

Comment:
This is correct from UpdateExtractor: <https://github.com/LandSandBoat/UpdateExtractor/blob/main/out/scripts/zones/Reisenjima/Text.lua.raw.txt#LL7518-L7518C24>

But we also turn anything in angle brackets back down into %: https://github.com/LandSandBoat/UpdateExtractor/blob/main/zone_texts.py#L81-L112

So this could be <item>

Reply:
Updated